lib/db.js

```diff
@@ -476,43 +467,43 @@ DB.prototype._createTable = function (keyspace, schema, tableName) {
     });
 
     // Finally, create the main data table
-    var cql = 'create table if not exists '
+    var sql = 'create table if not exists '
         + dbu.cassID(keyspace + '_' + tableName) + ' (';
     for (var attr in schema.attributes) {
         var type = schema.attributes[attr];
-        cql += dbu.cassID(attr) + ' ';
+        sql += dbu.cassID(attr) + ' ';
         switch (type) {
```

Comment:
Uptill now I was adding functionality in the order (create table, put, get, types) which is the order the tests are defined. But I agree with you that we should do the create table path properly and then move further.

I will update the patch with the types support first and then move further.